GliaWidgets/View/Chat/Entry/ChatMessageEntryView.swift

```diff
@@ -165,17 +164,13 @@ class ChatMessageEntryView: UIView {
         textView.autoPinEdge(toSuperviewEdge: .top, withInset: 13)
         textView.autoPinEdge(toSuperviewEdge: .bottom, withInset: 13)
         textView.autoPinEdge(toSuperviewEdge: .right, withInset: 8)
-        textView.autoAlignAxis(toSuperviewAxis: .horizontal)
+//        textView.autoAlignAxis(toSuperviewAxis: .horizontal)
```

Comment:
Can it be removed?

Reply:
fixed [9fbdd03](https://github.com/salemove/ios-sdk-widgets/pull/370/commits/9fbdd03de40296523e49e199fa156c61ff331ee1)